### ISSUE TITLE
.NETCore solution does not load correctly.

### ISSUE THREAD

**Issue by nopara73** [MetacoSA/NBitcoin#198]
After cloning the repo and clicking on `NBitcoin.NETCore.sln` I got this error message:  

> One or more projects in the solution were not loaded correctly.
> Please see the Output Window for details.

I click ok, then in the solution explorer under the `NBitcoin.NETCore.Tests` project:

> The project file or web cannot be found.

http://imgur.com/a/euNVK

Restoring the `NBitcoin` project and building it is successfull.
Although there are no sources under `NBitcoin` project in the solution explorer, but I think that is intentional, right? So you only use building the .NETCore project, not for developing, right?

Basically it looks for **`NBitcoin.NETCore.Tests\NBitcoin.NETCore.Tests.xproj`** file is missing, what is not under version control. It should solve the problem if you put push it.

**Comment by NicolasDorier** [comment]
I don't expect people to open NBitcoin.NETCore in VS. Use it in command line. Maybe I should remove the SLN.

**Comment by nopara73** [comment]
Either remove or add the .xproj file.  
Essentially it is a tradeoff. 
If you add the .xproj you are giving people the convenience who want to use their own version of NBitcoin with .NET Core.  
If you remove the .NET Core solution you are reducing the possibility of confusion for those who just want to participate.

**Comment by NicolasDorier** [comment]
I think xproj are being obsolete, and the json project file is being deprecated in favor of csproj recently (https://blogs.msdn.microsoft.com/dotnet/2016/10/19/net-core-tooling-in-visual-studio-15/)

Anyway, the sln should be removed

**Comment by nopara73** [comment]
Maybe this is what you mean, just not sure about your English:  
What the article says is this: project.json and .xproj will be obsolate in the future.  
**But they are not obsolate yet.**

**Comment by NicolasDorier** [comment]
I just nuked them c051f8c
